### DISCUSSION TITLE
List name length too small

### DISCUSSION THREAD

**Discussion by stiliajohny** [community/community#8996] [Other Feature Feedback, Questions, & Ideas]
Even though for most occasions the name is adequate, 
when using some emoji notations the list name is not long enough. 

As an example, if I use `:envelope_with_arrow:My list name here` it won't have enough characters 

Would it be considered to match the repository naming length constrains perhaps

**Comment by brianlovin** [comment]
I think the short term fix here is to use the emoji directly by copy/pasting it into the name field: 📩

**Reply by stiliajohny** [reply]
agree. that could be a midterm solution too. :)
